### ISSUE TITLE
update home page video

### ISSUE THREAD

**Issue by pdpinch** [mitodl/micromasters#4142]
waiting on URL from MicroMasters team

I think the current version is at https://github.com/mitodl/micromasters/blob/5d07dac5715ac11cc261727c75d2035709601307/cms/templates/cms/home_page.html#L86

**Comment by noisecapella** [comment]
Is this a duplicate of #4144?
